awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/VisualizerLink.jsx

```diff
@@ -94,7 +94,8 @@ function VisualizerLink({
         ];
 
   const handleLinkMouseEnter = () => {
-    ref.current.parentNode.appendChild(ref.current);
+    const startNode = document.getElementById('node-1');
+    ref.current.parentNode.insertBefore(ref.current, startNode);
```

Comment:
I think remember translation being tricky for workflow nodes (?) in the old ui - glad to have found an approach here :+1: